src/NServiceBus.Core/Receiving/ReceiveComponent.cs

```diff
@@ -174,7 +175,7 @@ partial class ReceiveComponent
                     var satelliteRecoverabilityExecutor = recoverabilityExecutorFactory.Create(satellite.RecoverabilityPolicy, satellite.ReceiveAddress);
 
                     await satellitePump.Initialize(satellite.RuntimeSettings, satellitePipeline.Invoke,
-                        satelliteRecoverabilityExecutor.Invoke, cancellationToken).ConfigureAwait(false);
+                        satelliteRecoverabilityExecutor.Invoke, (_, __) => Task.CompletedTask, cancellationToken).ConfigureAwait(false);
```

Comment:
Are we sure this is OK?

Reply:
Yes, we only trigger these events for the "main and instance receivers" not for the satellites